frontend/src/components/data.json

```diff
@@ -1,62 +1,70 @@
-[
+{
+  "Questions": [ 
     {
        "id": 0,
        "text": "Who are oss cameroon owners",
        "type": "Q",
        "children": [3]
     },
+    
     {
        "id": 1,
-       "text": "Sanix Darker, Elhmmn, Eric, Yvan, Sherlock Wisdom",
-       "type": "A",
-       "children": [2, 4]
-    },
-    {
-       "id": 2,
        "text": "About Oss Cameroon",
        "type": "Q",
        "children": [4, 5]
     },
+    
     {
-       "id": 3,
-       "text": "Open Source Community by Cameroonian developers",
-       "type": "A",
-       "children": []
-    },
-    {
-       "id": 4,
+       "id": 2,
        "text": "Goals",
        "type": "Q",
        "children": [6]
     },
     {
-      "id": 5,
-      "text": "Bring developers together, Cultivate the habit of contributing to open source projects in the Cameroonian developer ",
-      "type": "A",
-      "children": []
-    },
-    {
-      "id": 6,
+      "id": 3,
       "text": "Vision",
       "type": "Q",
       "children": []
     },
```

Comment:
If this is a Question `Q` it should have at least one child... because we can not have Questions witouth answers (maybe it's because you don't have yet appropriate children for this, in that case, you can ignore my comment).

Reply:
I think I should just remove the children element since I separated the questions from the answers?